src/main/resources/anniv5Resources/localization/eng/coresetpack/Keywordstrings.json

```diff
@@ -1,2 +1,9 @@
 [
+  {
+    "PROPER_NAME": "Synergy",
+    "NAMES": [
+      "synergy"
+    ],
+    "DESCRIPTION": "Additional effect if you have cards from 3 different packs in your hand."
```

Comment:
different is a little confusing here. Different would make me think it excludes itself which it doesn't do.

Reply:
would 'unique packs' alleviate this?